app/src/features/apiClient/screens/apiClient/components/modals/deleteApiRecordModal/DeleteApiRecordModal.tsx

```diff
@@ -59,6 +62,12 @@ export const DeleteApiRecordModal: React.FC<DeleteApiRecordModalProps> = ({ open
     setIsDeleting(false);
   };
 
+  const header = record.type === "api" ? "Delete API Request" : "Delete Collection";
+  const description =
+    record.type === "api"
+      ? `This action will permanently delete this API request. Are you sure you want to continue?`
+      : `This action will permanently delete the entire collection and its ${apiRequestCount} requests. Are you sure you want to continue?`;
+
```

Comment:
Same here, Use `RecordType` enum instead "api" string literals for record.type check